src/core_plugins/vega/public/data_model/es_query_parser.js

```diff
@@ -56,20 +57,39 @@ export class EsQueryParser {
     if (body === undefined) {
       url.body = body = {};
     } else if (!_.isPlainObject(body)) {
-      throw new Error('url.body must be an object');
+      throw new Error(i18n.translate('vega.esQueryParser.urlBodyValueTypeErrorMessage', {
+        defaultMessage: '{configName} must be an object',
+        values: { configName: 'url.body' },
+      }));
     }
 
     // Migrate legacy %context_query% into context & timefield values
     const legacyContext = url[LEGACY_CONTEXT];
     delete url[LEGACY_CONTEXT];
     if (legacyContext !== undefined) {
       if (body.query !== undefined) {
-        throw new Error(`Data url must not have legacy "${LEGACY_CONTEXT}" and "body.query" values at the same time`);
+        throw new Error(i18n.translate('vega.esQueryParser.dataUrlMustNotHaveLegacyAndBodyQueryValuesAtTheSameTimeErrorMessage', {
+          defaultMessage: 'Data url must not have legacy "{legacyContext}" and "{bodyQueryConfigName}" values at the same time',
+          values: {
+            legacyContext: LEGACY_CONTEXT,
+            bodyQueryConfigName: 'body.query',
+          },
+        }));
       } else if (usesContext) {
-        throw new Error(`Data url must not have "${LEGACY_CONTEXT}" together with "${CONTEXT}" or "${TIMEFIELD}"`);
+        throw new Error(i18n.translate('vega.esQueryParser.dataUrlMustNotHaveLegacyContextTogetherWithContextOrTimefieldErrorMessage', {
+          defaultMessage: 'Data url must not have "{legacyContext}" together with "{context}" or "{timefield}"',
+          values: {
+            legacyContext: LEGACY_CONTEXT,
+            context: CONTEXT,
+            timefiels: TIMEFIELD,
+          },
+        }));
       } else if (legacyContext !== true && (typeof legacyContext !== 'string' || legacyContext.length === 0)) {
-        throw new Error(`Legacy "${LEGACY_CONTEXT}" can either be true (ignores time range picker), ` +
-          'or it can be the name of the time field, e.g. "@timestamp"');
+        throw new Error(i18n.translate('vega.esQueryParser.legacyContextCanBeTrueErrorMessage', {
+          defaultMessage: 'Legacy "{legacyContext}" can either be true (ignores time range picker), \
```

Comment:
That way of multiline strings, was never standardized properly, why browsers behave differently. Some browser insert `\n` for that line break other don't. Can you please use either `'..' + '..'` still, or in case our system does not support this, put the whole string in one line.